web/docs/sizing.md

```diff
@@ -0,0 +1,57 @@
+##### Table of Contents
+
+[Introduction](#Introduction)  
+[8pt grid system](#8pt-grid-system)  
+[Pixels and Accessibility](#pixels-and-accessibility)
+
+## Introduction
+
+Documentation for sharing UX/UI design decisions for the ameliorate project.
+
+# 8pt grid system
+
+The 8pt Grid principle states that multiples of 8 (8, 16, 24, 32, 40, 48, 56, etc.) are used for the layout, dimension, pad, and margin of elements.
+
+## How to use it
+
+- Margin or Padding: 8px / 16px / 24px / 32px / 40px / 48px / 56px / ... / N\*8px
+- Iconography
+  - SVG: multiples of 8 ( 16×16, 24×24, 32×32, 40x40, ..., N\*(8x8) )
+- Typography
+  - font size: can vary between device.
+  - line height:
+    - multiples of 8 ( 8, 16, 24, 32, 40, 48, ..., N\*8 )
+    - multiples of 4 ( 4, 8, 12, 16, 20, 24, …, N\*4 )
+- !important treat this as a strong suggestion, flexibility is key.
+
+## Why?
+
+- It provides a visual hierarchy to elements and drives consistent scalability with fewer decisions.
+- Helps maintain a consistent feel across the website.
+- Helps designers and developers work faster on a project.
+- Various device screen sizes are multiples of 8.
+- Helps build responsive websites for various devices.
+
+Sources:
+
+- [Everything you should know about 8 point grid system in UX design](https://uxplanet.org/everything-you-should-know-about-8-point-grid-system-in-ux-design-b69cb945b18d?gi=d321e6edb7fd)
+- [What Is 8 Point Grid System In UX Design](https://fronty.com/what-is-8-point-grid-system-in-ux-design/)
+- [The 8-Point Grid](https://spec.fm/specifics/8-pt-grid)
+
+# Pixels and Accessibility
+
+## How to use it
+
+- Use `rem` scales with the user's default browser font size.
+- If converting it to `rem` adds visual clarity or functionality, do it. Otherwise keep it fixed.
+- If you are unsure, test it. Change the defaults: [Chrome](https://support.microsoft.com/en-us/microsoft-edge/increase-default-text-size-in-microsoft-edge-c62f80af-381d-0716-25a3-c4856dd3806c), [FireFox](https://support.mozilla.org/en-US/kb/change-fonts-and-colors-websites-use), [Safari](https://support.apple.com/en-gb/HT207209), [Edge](https://support.microsoft.com/en-us/microsoft-edge/increase-default-text-size-in-microsoft-edge-c62f80af-381d-0716-25a3-c4856dd3806c).
+- There is no cookie cutter method, it will take some intuition.
```

Comment:
```suggestion
# Pixels and Accessibility

Using `rem` scales sizes with the user's default browser font size. [mention that we're summarizing from the source]

## How to use it

- Generally: use `px` for aesthetics, use `rem` for visual clarity or functionality.
- There is no cookie cutter method to "correctly" pick between them, just try your best.
- It can help to test what the sizing looks like at different default font sizes. Change the defaults: [Chrome](https://support.microsoft.com/en-us/microsoft-edge/increase-default-text-size-in-microsoft-edge-c62f80af-381d-0716-25a3-c4856dd3806c), [FireFox](https://support.mozilla.org/en-US/kb/change-fonts-and-colors-websites-use), [Safari](https://support.apple.com/en-gb/HT207209), [Edge](https://support.microsoft.com/en-us/microsoft-edge/increase-default-text-size-in-microsoft-edge-c62f80af-381d-0716-25a3-c4856dd3806c).
```